vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -137,6 +129,28 @@ def _filter_interaction(
 
         return data_frame
 
+    @staticmethod
+    def _optimise_fig_layout_for_dashboard(fig):
```

Comment:
@antonymilne and I decided to put it inside the graph.__call__ for now. If at some point there will be more overwrites such that it's difficult to keep track, we can create separate theme and merge it to the existing vizro_dark/vizro_light.